### DISCUSSION TITLE
Problem watch face maker

### DISCUSSION THREAD

**Discussion by ricardfs** [zepp-health/zeppos-forum#23] [Developer Feedback]
I don't know if the only one who is experiencing illogical problems with watchface maker, these days when I try to upload the different elements of images it does not accept them, at first the image appears but then it disappears, when it seems that everything works I scan the qr code to try on the device, bip u pro, and some of the elements do not appear, does anyone else have problems creating watchfaces?

**Comment by dmitrykurteanu** [comment]
I can't upload any image too. Once the upload bar is full, the uploaded image quickly flashes and then I get a "broken image" icon and if I want to preview it, the preview is emtpy.

**Reply by ricardfs** [reply]
That's right, we have the same problem, I've tried to create a watch face for several different devices and I get the same error on all of them, I imagine it's a general platform problem.

**Reply by MyoungXUE** [reply]
Our colleagues have checked the logs, and the problem of uploading images is caused by the downtime of the cloud service, it is an occasional situation.

We are very sorry for the trouble caused to you.

**Reply by dmitrykurteanu** [reply]
Thanks for the info! Any estimates on when will this be fixed?

**Reply by MyoungXUE** [reply]
> Thanks for the info! Any estimates on when will this be fixed?

Please try again, still can't upload images?

This is an intermittent problem with the cloud service provider, the probability is very small, we suggest trying again after a period of time. We have given them feedback on the problem, and hope to improve the stability of the follow-up service

**Reply by dmitrykurteanu** [reply]
> > Thanks for the info! Any estimates on when will this be fixed?
> 
> Please try again, still can't upload images?
> 
> This is an intermittent problem with the cloud service provider, the probability is very small, we suggest trying again after a period of time. We have given them feedback on the problem, and hope to improve the stability of the follow-up service

Yes, it still doesn't work. Every time I upload an image, it shows a "broken image" icon and it won't work and won't preview. If I wait 6-8 hours, the image eventually shows up and starts working, but I can't wait 8 hours every time I want to upload an image.

**Reply by dmitrykurteanu** [reply]
Does it work for you? Is this a regional thing?

**Reply by ricardfs** [reply]
It still doesn't work for me, the broken image icon keeps appearing, like my partner, if I wait a few hours the image appears but in my case if I scan the qr code the image does not appear on the device.

**Reply by MyoungXUE** [reply]
We are very sorry for the unavailability of the function, I have contacted the relevant colleagues to check again, please be patient for a while.

**Comment by ricardfs** [comment]
Today I have managed to finish a watch face completely, it was very difficult for me to accept the images, but to my surprise when I look at the preview everything is fine, when I pass it to the device with the qr code I see that it has eliminated all the transparent areas, transparent areas on the device are now a solid black background.

what's going on?

**Reply by sunnyBob** [reply]
This may be caused by image format. If your image is transparent, you can select "solid and Transparent".If your image is gradient, you can select "Gradient". And if your image is not transparent and gradient, you can select "solid" format.
<img width="368" alt="image" src="https://user-images.githubusercontent.com/18245566/208597517-844d390f-57b0-4520-8ba4-81ad98d53877.png">

**Comment by MyoungXUE** [comment]
Sorry to wait, I just noticed the problem and will contact the relevant colleagues to contact you.

**Reply by ricardfs** [reply]
Thank you for your answer, at least now we know what the problem is, we hope it will be resolved soon

**Comment by MiroslavGubenko** [comment]
Why problem is still not fixed? Any images for any widgets are not uploading. Bg upliading , but need changing size pic. Maybe, are you know other programs for making watch face? (for GTR2)
![2023-02-13_14-34](https://user-images.githubusercontent.com/38442573/218458958-88680cb7-9d9b-4067-95ed-60d10610d3f3.png)

**Reply by ricardfs** [reply]
It has not been updated for a long time, with the GTS model it works well, you can try it with the GTR model.
https://github.com/ricardfs/GTR2_editor

**Reply by MiroslavGubenko** [reply]
But, thank you for trying to help! Situation is so sad for me. Any editor doesn't work whith GTR 2.
![2023-02-13_19-30](https://user-images.githubusercontent.com/38442573/218530288-5936830b-ea52-4186-b5e6-e28345e294f8.png)

PS. How to impoting JSON img? Where I can find info about this editor?

**Reply by dmitrykurteanu** [reply]
I don't think it's related to any specific model. It doesn't work for any model in my case. Can't upload any image whatsoever.
I had the same issue a month ago, and it was region related (would work with VPN for certain countries, not with others). It was "fixed" at some point, but it's not working again, and VPN doesn't help.

**Reply by MiroslavGubenko** [reply]
![2023-02-14_09-08](https://user-images.githubusercontent.com/38442573/218665024-c95eccab-dec3-4ad4-99fc-0faa1d2cbe83.png)
Haha. Today is working. i try to use vpn and that work. And....same without vpn. "Awesome" service.

**Reply by ricardfs** [reply]
Still not working for me, even with VPN

**Reply by seldish-og** [reply]
hey
